Fakebook.Profile/Fakebook.Profile.DataAccess/ProfileRepository.cs

```diff
@@ -164,7 +232,7 @@ public async Task CreateProfileAsync(DomainProfile profileData)
             // have to have this try catch block to prevent errors from data base
             try
             {
-                var newUser = ToEntityProfile(profileData); // convert
+                var newUser = await ToEntityProfile(profileData); // convert
```

Comment:
like i said if we need to await it, it should normally have async suffix.

Reply:
Renamed